docs/quickstart/generate-report-ui.mdx

```diff
@@ -120,12 +120,12 @@ The command will use the provided connection profile to access the data warehous
 
 ---
 
-## Send Report to Slack
+## Send Report to External Platform
```

Comment:
You can make a 'Sharing the report' snippet. 
(I explained on the other docs branch you created)